pkgs/development/python-modules/pywick/default.nix

```diff
@@ -68,6 +72,5 @@ buildPythonPackage rec {
     homepage = "https://github.com/achaiah/pywick";
     license = lib.licenses.mit;
     maintainers = with lib.maintainers; [ bcdarwin ];
-    broken = true;  # Nixpkgs missing `albumentations` and `prodict`
```

Comment:
`pywick` seems a bit unmaintained.  I don't use it any longer, so we might consider dropping it if you aren't using it either?

Reply:
I thought I could leave it as long as it works, but would you like me to remove it?
I agree with you, as it is unlikely to be maintained in the future.

Reply:
Up to you; if not, I'll probably remove it later.

Reply:
Thanks, I will remove it.